app/controllers/idv/gpo_verify_controller.rb

```diff
@@ -51,21 +51,22 @@ def create
         return
       end
 
-      if result.extra[:pending_in_person_enrollment]
-        redirect_to idv_in_person_ready_to_verify_url
-      else
-        prepare_for_personal_key
+      prepare_for_personal_key
 
-        redirect_to idv_personal_key_url
-      end
+      redirect_to idv_personal_key_url
     end
 
     private
 
+    def pending_in_person_enrollment?
+      return false unless IdentityConfig.store.in_person_proofing_enabled
+      current_user.pending_in_person_enrollment.present?
+    end
+
     def prepare_for_personal_key
       event, _disavowal_token = create_user_event(:account_verified)
```

Comment:
Two questions that might be unhelpful:
1. Do we need to destructure `_disavowal_token` since we don't use it?
2. I'm not super familiar with `create_user_event`, but are there side effects to calling it? The previous code used to completely skip this during IPP, so I just want to make sure nothing unexpected happens. 

Otherwise this looks good!

Reply:
Good catch. I'll move that line into the unless statement below.